src/FunctionInvocationUnary.cpp

```diff
@@ -224,13 +229,28 @@ FunctionInvocationUnary::Output(std::ostream &out) const
 					if (CGOptions::identify_wrappers()) {
 						out << ", " << id;
 					}
-					out << ")";
+
+					out << "/*unary wrapper*/)";
```

Comment:
Can we remove the comment? Or, emitting it only under certain condition, e.g., ```CGOptions::float_test()```.